ddtrace/contrib/asgi/middleware.py

```diff
@@ -169,7 +169,13 @@ async def wrapped_send(message):
                 span, self.integration_config, status_code=status_code, response_headers=response_headers
             )
 
-            return await send(message)
+            try:
+                return await send(message)
+            finally:
+                # Per asgi spec, "more_data" is used if there is still data to send
```

Comment:
Do we mean to say `"more_body"` here? I can't find any mention of `"more_data"` in the asgi spec, and it looks like we care about `"more_body"`, but I could definitely be missing it.

Reply:
You're right! Thanks for catching that! I updated the comment to fix this. Marking as resolved.